src/GovUk.Education.ExploreEducationStatistics.Data.Services.Tests/SubjectMetaServicePermissionTests.cs

```diff
@@ -103,48 +111,89 @@ await PolicyCheckBuilder<DataSecurityPolicies>()
                         SubjectId = SubjectId
                     };
 
-                    var cancellationToken = new CancellationTokenSource().Token;
-
-                    var releaseSubjectRepository = new Mock<IReleaseSubjectRepository>(Strict);
+                    var release = new ContentRelease
+                    {
+                        Id = ReleaseId,
+                        Published = DateTime.UtcNow.AddDays(-1)
+                    };
 
-                    releaseSubjectRepository.Setup(mock => mock.GetReleaseSubjectForLatestPublishedVersion(SubjectId))
-                        .ReturnsAsync(ReleaseSubject);
+                    await using var contextDbContext = InMemoryContentDbContext();
+                    await contextDbContext.Releases.AddAsync(release);
+                    await contextDbContext.SaveChangesAsync();
 
+                    await using var statisticsDbContext = StatisticsDbUtils.InMemoryStatisticsDbContext();
+                    await statisticsDbContext.ReleaseSubject.AddAsync(ReleaseSubject);
+                    await statisticsDbContext.SaveChangesAsync();
+                    
                     var service = SetupSubjectMetaService(
-                        releaseSubjectRepository: releaseSubjectRepository.Object,
-                        userService: userService.Object
-                    );
+                        userService: userService.Object, 
+                        statisticsDbContext: statisticsDbContext, 
+                        contentDbContext: contextDbContext);
+
+                    var cancellationToken = new CancellationTokenSource().Token;
 
                     return await service.FilterSubjectMeta(null, query, cancellationToken);
                 }
             );
     }
 
+    [Fact]
+    public async Task GetReleaseSubjectForLatestPublishedVersion()
+    {
+        // Note that there are on explicit security policy checks enforced for this method.
```

Comment:
Small grammar issue here I think?

Reply:
Typo - should've read "no", not "on" :)  Cheers!